### ISSUE TITLE
Cover page: Table of Contents accessibility improvements

### ISSUE THREAD

**Issue by jhung** [pressbooks/pressbooks-book#175]
### Prerequisites

* [x] Can you reproduce the behavior you wish to address with [all plugins except Pressbooks deactivated](http://codex.wordpress.org/Multisite_Network_Administration#Plugins)?
* [x] Are you running the [latest version of Pressbooks](http://github.com/pressbooks/pressbooks/releases/latest/), the latest version of [Pressbooks Book](http://github.com/pressbooks/pressbooks-book/releases/latest/), the latest version of [this book theme](http://github.com/pressbooks/pressbooks-book/releases/latest/) and the [latest version of WordPress](http://codex.wordpress.org/Upgrading_WordPress)?
* [x] Are you experiencing the issue while using this book theme?
* [x] Did you [perform a cursory search](https://github.com/issues?q=+is%3Aissue+repo%3Apressbooks%2Fpressbooks-book) to see if your bug or enhancement is already listed?

### Description

The Table of Contents located on the cover page can be improved for accessibility.

* The table of contents should be enclosed in a `<nav>` element.
* The toggle triangle SVG should have `role="image"` attribute.
* The `<h3>` elements in the table of contents may not be necessary as the list structure already conveys hierarchy. Also using `<h3>` in a list is unusual - someone using a screen reader to navigate by headers does not normally expect to land in a list.

Alternatively, the table of contents can be split into separate lists for improved usability and accessibility (instead of a single long nested list).

Example:

```html
<h3><button>Front Matter</button></h3>
<nav>
<ol>
<li> Item 1</li>
<li> etc. </li>
</ol>
</nav>

<h3><button>Main body</button></h3>
<nav>
<ol>
<li> Item 1</li>
<li> etc. </li>
</ol>
</nav>
```

**Comment by greatislander** [comment]
@jhung Did you mean [`role="img"`](https://www.w3.org/TR/wai-aria-1.1/#img)?
